liquidhaskell-boot/src/Language/Haskell/Liquid/Bare/Measure.hs

```diff
@@ -401,6 +403,21 @@ getDefinedSymbolsInLogic env measEnv specs =
     localize :: F.Symbol -> F.LocSymbol
     localize sym = maybe (dummyLoc sym) varLocSym $ L.lookup sym (Bare.reSyms env)
 
+-- Get the set of `DataCon`s (DCs) needed for the reflection of a given list of variables,
+-- and which are not already present in the logic
+getReflDCs :: Bare.MeasEnv -> [Ghc.Var] -> S.HashSet Ghc.DataCon
+getReflDCs measEnv vars = dcsUndefinedInLogic
+  where
+    -- List of wired DCs that cannot be found in the measure environment yet are already defined/should not be
+    -- reflected. Written as a list of symbols because that's easier than trying to get the corresponding DCs from GHC.
```

Comment:
```suggestion
    -- List of wired DCs that cannot be found in the measure environment as they are
    -- eliminated in translation from core.
    -- Written as a list of symbols because that's easier than trying to get the corresponding DCs from GHC.
```